src/client/routes/Market/index.tsx

```diff
@@ -78,7 +79,10 @@ export const Market = () => {
       orderDirection: 'desc',
     },
   };
-  const fetchMarketData = () => dispatch(LinesActions.getLines(defaultLineCategories));
+  const fetchMarketData = () =>
+    dispatch(TokensActions.getTokens())
+      .then((res) => dispatch(LinesActions.getLines(defaultLineCategories)))
+      .then((error) => dispatch(LinesActions.getLines(defaultLineCategories)));
```

Comment:
why refetch on error without changing params? What do we expect to change?

Reply:
yea this is dumb, I don't know why I did this. 

replaced with this: 
```  const fetchMarketData = () =>
    dispatch(TokensActions.getTokens()).then(() => dispatch(LinesActions.getLines(defaultLineCategories)));
```